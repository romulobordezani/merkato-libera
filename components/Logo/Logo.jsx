import Link from 'next/link';
import React from 'react';

import styles from './Logo.module.scss';

const Logo = () => (
  <div className={styles.logo}>
    <Link href="/">
      <img
        className={styles.logo_img}
        src="/images/Logo_ML.png"
        srcSet="/images/Logo_ML@2x.png.png 1000w, /images/Logo_ML@2x.png.png  2000w"
        alt="Mercado Livre"
      />
    </Link>
  </div>
);

export default Logo;
