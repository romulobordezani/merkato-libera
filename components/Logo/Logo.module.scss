.logo {
  width: 60px;
  margin-right: 20px;
  cursor: pointer;

  &_img {
    width: 100%;
    height: auto;
  }
}
