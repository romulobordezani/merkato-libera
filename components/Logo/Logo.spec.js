import React from 'react';
import renderer from 'react-test-renderer';

import Logo from './Logo';

describe('Logo', () => {
  it('Snapshot renders default', () => {
    expect(renderer.create(<Logo />)).toMatchSnapshot();
  });
});
