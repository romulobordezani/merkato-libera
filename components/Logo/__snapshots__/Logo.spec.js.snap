// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Snapshot renders default 1`] = `
<div>
  <img
    alt="Mercado Livre"
    onClick={[Function]}
    onMouseEnter={[Function]}
    src="/images/Logo_ML.png"
    srcSet="/images/Logo_ML@2x.png.png 1000w, /images/Logo_ML@2x.png.png  2000w"
  />
</div>
`;
